README.md

```diff
@@ -17,10 +18,10 @@ ES6 classes make it easy to define custom errors by subclassing `Error`. Subclas
 ```js
 const OError = require('@overleaf/o-error')
 
-function doSomethingBad () {
+function doSomethingBad() {
   throw new OError({
     message: 'did something bad',
-    info: { thing: 'foo' }
+    info: { thing: 'foo' },
```

Comment:
This does feel like a departure from our existing style. Is this part of the prettier update? I assumed it was standard until I noticed that we're not using that

Reply:
Yep: https://prettier.io/docs/en/options.html#trailing-commas